device_operation/device_gemm_xdl_instance_f32_f32_f32_km_kn_mn.cpp

```diff
@@ -18,6 +19,24 @@ template <ck::index_t... Is>
 using S = ck::Sequence<Is...>;
 
 // Compilation parameters for a[k, m] * b[k, n] = c[m, n]
+#if CK_USE_SPLITK_XDLOPS
```

Comment:
need to add split-k gemm in a separate file

Reply:
done